projects/1.18/assets/laser-mod/lasermod/lang/zh_cn.json

```
@@ -0,0 +1,78 @@
+{
+	"block.lasermod.laser": "激光",
+	"block.lasermod.laser_catcher": "激光传感器",
+	"block.lasermod.infuser": "注能器",
+	"block.lasermod.mod_station": "改造站",
+	"block.lasermod.laser_projector": "激光投影仪",
+	"block.lasermod.laser_controller": "激光控制器",
+	"block.lasermod.mirror": "镜子",
+	"block.lasermod.advanced_laser": "高级激光（§4WIP§r）",
+	"block.lasermod.precision_assembler": "模块装配台",
+	"block.lasermod.device_hub": "激光集束器（§4WIP§r）",
+
+	"item.lasermod.laser_crystal": "激光晶体",
+	"item.lasermod.laser_tool_base": "激光工具外壳",
+	"item.lasermod.ir_glasses": "红外眼镜",
+	"item.lasermod.linker": "连接器",
+	"item.lasermod.silicon_plate": "硅制升级模板",
+	"item.lasermod.printed_circuit_board": "电路板",
+	"item.lasermod.silicon": "硅",
+	"item.lasermod.manual": "手册",
+
+	"item.lasermod.laser_pickaxe": "激光镐",
+	"item.lasermod.laser_sword": "激光剑",
+	"item.lasermod.laser_shovel": "激光锹",
+	"item.lasermod.laser_axe": "激光斧",
+	"item.lasermod.laser_hoe": "激光锄",
+	"item.lasermod.laser_multitool": "激光全能工具",
+
+	"item.lasermod.laser_helmet": "激光头盔",
+	"item.lasermod.laser_chestplate": "激光胸甲",
+	"item.lasermod.laser_leggings": "激光护腿",
+	"item.lasermod.laser_boots": "激光靴子",
+
+	"item.lasermod.upgrade_speed": "速度升级模块",
+	"item.lasermod.upgrade_mining": "采矿升级模块",
+	"item.lasermod.upgrade_fire": "火焰升级模块",
+	"item.lasermod.upgrade_color": "色彩升级模块",
+	"item.lasermod.upgrade_mode": "模式升级模块",
+	"item.lasermod.upgrade_damage": "伤害升级模块",
+	"item.lasermod.upgrade_no_damage": "无害升级模块",
+	"item.lasermod.upgrade_silence": "静音升级模块",
+	"item.lasermod.upgrade_pull": "拉升级模块",
+	"item.lasermod.upgrade_push": "推升级模块",
+	"item.lasermod.upgrade_distance": "距离升级模块",
+	"item.lasermod.upgrade_airtight_seal": "气密升级模块",
+	"item.lasermod.upgrade_capacity": "容量升级模块",
+
+	"item.upgrade.tooltip": "可用于：",
+	"item.upgrade.tooltip.tier": "等级 %s",
+	"item.upgrade.tooltip.laser_tool": "激光工具",
+	"item.upgrade.tooltip.laser_helmet": "激光头盔",
+	"item.upgrade.tooltip.laser_chestplate": "激光胸甲",
+	"item.upgrade.tooltip.laser_leggings": "激光护腿",
+	"item.upgrade.tooltip.laser_boots": "激光靴子",
+	"item.upgrade.tooltip.laser_armor": "激光装甲",
+
+	"container.lasermod.laser": "激光",
+	"container.lasermod.infuser": "注能器",
+	"container.lasermod.precision_assembler": "模块装配台",
+
+	"container.lasermod.laser.button.options": "选项",
+	"container.lasermod.laser.red": "红(R)",
+	"container.lasermod.laser.green": "绿(G)",
+	"container.lasermod.laser.blue": "蓝(B)",
+
+	"death.block.laser": "%s已被激光烧毁",
+	"death.armor.laser": "%1$s已被%2$s的激光护甲烧毁",
+
+	"itemGroup.tabLaserMod": "激光模组",
+	"itemGroup.tabLaserMod.upgrades": "激光升级模块",
+
+	"key.categories.lasermod": "激光模组",
+	"lasermod.armor_toggle": "装甲切换",
+
+	"subtitle.lasermod.laser_activate": "激光激活",

```

```suggestion
	"subtitle.lasermod.laser_activate": "激光：激活",
```
https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95
此类问题请全文通查